scripts/deploy/govcms-config-backup

```diff
@@ -32,4 +32,5 @@ drush config:export sync -y --destination "$GOVCMS_BACKUP_DIR/config"
 tar -czf "$GOVCMS_BACKUP_DIR/$GOVCMS_CONFIG_BACKUP.tar.gz" -C "$GOVCMS_BACKUP_DIR/config" --remove-files
 
 echo "[info]: Saved to $GOVCMS_BACKUP_DIR/$GOVCMS_CONFIG_BACKUP.tar.gz"
+
```

Comment:
here and below - success message is a last code line and has exactly one empty line above it.